source/brailleDisplayDrivers/freedomScientific.py

```diff
@@ -287,28 +296,32 @@ def _onReceive(self, data):
 		"""
 		if self.isUsb:
 			data = BytesIO(data)
-			packetType = data.read(1)
+			packetType: bytes = data.read(1)

```

Comment:
Since this is just one byte, do we have a better type declaration for that?

Reply:
No, I don't think so. You could do `data.read(1)[0]` or `ord(data.read(1))` to take it as an int. Considering we then compare to several bytes values, it's best not to change the type. You can't compare bytes to int values directly.